configuration/arm64/mysql/add_v2xhub_user.bash

```diff
@@ -3,7 +3,12 @@
 #set -e
 
 # Ensure mysql-client is installed
-REQUIRED_PKG="mysql-client"
+arch=$(dpkg --print-architecture)
+if [ $arch = "amd64" ]; then
```

Comment:
Why are we checking for amd64 architecture for the arm64 configuration?  I would expect that this configuration is only used on arm architectures.  Maybe the check was supposed to be arm64?

Reply:
Ah, my bad. The idea at the moment was to merge both configurations into one folder, but I couldn't figure out how to get different docker images pulled from one docker-compose.yml file. Probably better to keep them as is.